code/modules/core_implant/cruciform/rituals/group.dm

```diff
@@ -145,7 +145,7 @@
 	effect_type = /datum/group_ritual_effect/cruciform/crusade
 
 /datum/group_ritual_effect/cruciform/crusade/success(var/mob/living/M, var/cnt)
-	if(cnt < 6)
+	if(cnt < 3) //Eclipse edit - FOUR PEOPLE IS STIL A CRUSADE
```

Comment:
Three people is enough for a crusade, not four, according to the code.

Reply:
It's four. The Preacher/Inquisitor are not counted in their rituals.